xbmc/music/Artist.cpp

```diff
@@ -215,16 +215,11 @@ bool CArtist::Save(TiXmlNode *node, const std::string &tag, const std::string& s
   for (const auto& it : discography)
   {
     // add a <album> tag
-    TiXmlElement cast("album");
-    TiXmlNode *node = artist->InsertEndChild(cast);
-    TiXmlElement title("title");
-    TiXmlNode *titleNode = node->InsertEndChild(title);
-    TiXmlText name(it.first);
-    titleNode->InsertEndChild(name);
-    TiXmlElement year("year");
-    TiXmlNode *yearNode = node->InsertEndChild(year);
-    TiXmlText name2(it.second);
-    yearNode->InsertEndChild(name2);
+    TiXmlElement discoElement("album");
+    TiXmlNode *node = artist->InsertEndChild(discoElement);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types